redisinsight/api/src/modules/browser/redisearch/redisearch.service.ts

```diff
@@ -269,6 +271,34 @@ export class RedisearchService {
     }
   }
 
+  public async deleteIndex(
+    clientMetadata: ClientMetadata,
+    dto: IndexDeleteRequestBodyDto,
+  ): Promise<void> {
+    const { index } = dto;
+    this.logger.debug(`Deleting redisearch index ${index}.`, clientMetadata);
```

Comment:
Let's not add index into log message string

Reply:
tl;dr - removed, thanks for your input.

As discussed over chat, my intention was to provide some helpful clues in case we have to debug issues related to that feature, and with the intention that there is some chance that the index name doesn't categorize fully as "user data". I tried to move it as an n-th parameter, not part of the message itself, but there is another issue with the fact we work with buffers and the index name is not always a string. 

So, as a conclusion, in order to avoid any potential issues, I have removed it completely from the logs.